src/AlertBox2/AlertBox2.less

```diff
@@ -42,15 +41,27 @@
 
 .AlertBox2--contentContainer {
   .flexbox();
-  .items--center();
-  .flex--grow();
+  .items--baseline();
+}
+
+.AlertBox2--contentContainer--withButtons {
+  line-height: @size_xl;
 }
 
 .AlertBox2--content {
-  .flex--grow();
   .text--semi-bold();
   color: @neutral_dark_gray;
   .margin--left--m();
+
+  & p {
+    .text--regular();
+    .margin--top--2xs();
```

Comment:
I think I understand the rationale as to why we want to update the text for `<p>` elements but wondering if there are any concerns around potentially adding top margin for every paragraph element vs only doing so for direct descendants (i.e. `& > p`)

Reply:
I imagine this shouldn't be an issue if we conform to the rule of only having a single line for an alert but  I can see a world where a non-direct child element might not need this extra spacing (e.g. a modal that is being opened within the alert but is fixed to the page rather than the alert box) and could be confusing while developing

Reply:
oh yeah, thanks. I can leave it up to the caller to specify spacing 👍